oracle/src/events/processTransfers/index.js

```diff
@@ -13,12 +13,25 @@ const limit = promiseLimit(MAX_CONCURRENT_EVENTS)
 function processTransfersBuilder(config) {
   const { bridgeContract, web3 } = config.home
 
-  const userRequestForAffirmationAbi = config.foreign.bridgeABI.find(
-    e => e.type === 'event' && e.name === 'UserRequestForAffirmation'
-  )
-  const tokensSwappedAbi = config.foreign.bridgeABI.find(e => e.type === 'event' && e.name === 'TokensSwapped')
```

Comment:
Do we want to remove special handling for events related to DSR? Imagine a situation when some bridge validators would like to run the oracles by observing all events starting from 2018 will the transfers related to DSR handled properly and discarded?

Reply:
No, it should still work. We have this clause for that - https://github.com/poanetwork/tokenbridge/blob/07336284c612f255d3b9a70e3012d682dd0dc25d/oracle/src/events/processTransfers/index.js#L66-L69

Reply:
Great! thanks